Webpack config file.

jennylia opened the pull request:
Making dist webpack file sourcemap debuggable for end users who build on
top of react-storybook.

This code review addresses issues of #457
Since the react-storybook is used as a toolkit for developers to build custom react components on top of it, changing this option allows a relatively light weight debug friendly version for other developers to build and debug their separate components built/demoed on react-storybook.

jennylia committed:
Webpack config file.

Making dist webpack file sourcemap debuggable for end users who build on
top of react-storybook.

mnmtanish commented:
Hi @jennylia 
The `dist` directory contains transpiled code. Can you change files in the src directory:
https://github.com/kadirahq/react-storybook/tree/master/src/server/config

jennylia committed:
Revert "Webpack config file."
This change should not be in the dist directory as mnmtanish suggested.

This reverts commit ecc30d801af3b205997b6803a632daee04af649a.

jennylia committed:
adding inline-source-map to devtools of webpack.config

jennylia commented:
@mnmtanish Thank you for your comment.
I have reverted my previous commit and added a commit that changes in the webpack config file. I have tested this against "npm run prepublish" and verified that it indeed change the webpack.config file for the dist directory.

arunoda commented:
We covered this with: https://github.com/kadirahq/react-storybook/pull/577

arunoda commented:
Thanks for the PR.
